_data-prepper/logs.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Log4j configuration
+nav_order: 12
+---
+
+#Log4j configuration
+
+The following describes how Data Prepper performs logging.
+
+# Logging 
+Data Prepper uses [SLF4J](http://www.slf4j.org/) with a [Log4j 2 binding](http://logging.apache.org/log4j/2.x/log4j-slf4j-impl/). 
+
+For Data Prepper 2.0 or above, Log4j 2 configuration file can be found and edited in `config/log4j2.properties` in the application's home directory. Default properties for Log4j 2 can be found in `log4j2-rolling.properties` in the *shared-config* directory.
+
+For Data Prepper before version 2.0, Log4j 2 configuration can be overridden by setting "log4j.configurationFile" system property when running Data Prepper. Default properties for Log4j 2 can be found in `log4j2.properties` in the *shared-config* directory. 
```

Comment:
"... by setting the "log4j.configurationFile" system property ..." 
I believe a system property is set in a configuration file and we would express this as code. If that's right, then it should be set as code: `log4j.configurationFile`.
Also, if this is a property, is it Boolean? In other words, would it be accurate to say:
"... by setting the `log4j.configurationFile` system property to `true`."?

Reply:
@oeyh Can you confirm that saying "... by setting the `log4j.configurationFile` system property to `true`" is accurate here?

Reply:
It's not like that. Please refer to the example command following the above text:

```
java "-Dlog4j.configurationFile=config/custom-log4j2.properties" -jar data-prepper-core-$VERSION.jar pipelines.yaml data-prepper-config.yaml
```

This is how `log4j.configurationFile` property is set when running data prepper, with `-D{property_name}={property_value}`. The value here is a path to the log4j2 config file.

Reply:
@oeyh Would this work? "The following command is how the `log4j.configurationFile` property is set when running Data Prepper.  This property uses `-D{property_name}={property_value}` as the value, which is a path to the Log4j2 configuration file."

Reply:
@chenqi0805 Would this work? "The following command is how the log4j.configurationFile property is set when running Data Prepper. This property uses -D{property_name}={property_value} as the value, which is a path to the Log4j2 configuration file."

Reply:
How about making it more concise by 
```
...can be overridden by setting system property "-Dlog4j.configurationFile={property_value}" when running Data Prepper, where {property_value} is a path to the Log4j2 configuration file.
```